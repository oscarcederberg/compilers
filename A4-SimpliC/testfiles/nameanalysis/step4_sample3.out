Error at line 6: symbol 'gcd1' is not declared
Error at line 7: symbol 'gcd2' is not declared
Error at line 8: symbol 'gcd1' is not declared
Error at line 8: symbol 'gcd2' is not declared
