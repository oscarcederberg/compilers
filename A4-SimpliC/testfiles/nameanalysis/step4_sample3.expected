Error at line 6: symbol 'gcd1' has not been declared before this use!
Error at line 7: symbol 'gcd2' has not been declared before this use!
Error at line 8: symbol 'gcd1' has not been declared before this use!
Error at line 8: symbol 'gcd2' has not been declared before this use!
